packages/network/src/api/image/requests.ts

```diff
@@ -83,15 +102,65 @@ export interface Add2ShotCloseUpImageOptions {
   compliance?: ComplianceOptions;
 }
 
+/**
+ * Options specififed when adding a close up (an "add damage" image) to an inspection using the 2-shot process.
```

Comment:
The TSDocs is not matching the interface; it should mention video frame instead, right?